editor/plugins/org.fusesource.ide.project/src/org/fusesource/ide/project/camel/CamelFacetInstallationDelegate.java

```diff
@@ -0,0 +1,245 @@
+/*******************************************************************************
+ * Copyright (c) 2011 Red Hat, Inc.
```

Comment:
Is it a moved file that github wasn't able to find? or a new one? in this the copyright date is not correct